projects/smart-table/src/lib/smart-table/smart-table.component.ts

```diff
@@ -322,6 +323,8 @@ export class SmartTableComponent implements OnInit, OnDestroy {
       this.pageSize$,
       this.currentPage$
     ).pipe(
+      // during initial configuration several values get pushed
+      debounceTime(50),
```

Comment:
If the purpose of this is to 'wait' for several values to pass through, please use skip()